CHANGELOG.md

```diff
@@ -6,6 +6,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [4.1.0]
+### Added
+- Add interactive web UI for selecting package versions to release ([#166](https://github.com/MetaMask/create-release-branch/pull/166))
+  - Added `--interactive` (`-i`) flag to launch a web-based UI for easier version selection
+  - Added `--port` option to configure the web server port (default: 3000)
+
+### Changed
+- Refine breaking change dependent detection to only consider peer dependencies ([#170](https://github.com/MetaMask/create-release-branch/pull/170))
+  - This change supports our policy of requiring packages with breaking changes to be released alongside their dependents
+  - Regular dependencies are no longer included in this check
```

Comment:
It looks like we are missing an entry for #158. This was added below in 3.1.0 (which was a backport) but I believe it would also appear in this release too:

``` suggestion
  - Regular dependencies are no longer included in this check
- Allow `npm:name@version` dependency redirections in manifest ([#158](https://github.com/MetaMask/create-release-branch/pull/158))
```